android/filament-android/src/main/cpp/IndexBuffer.cpp

```diff
@@ -91,8 +91,7 @@ Java_com_google_android_filament_IndexBuffer_nSetBuffer(JNIEnv *env, jclass type
 
     BufferDescriptor desc(data, sizeInBytes, &JniBufferCallback::invoke, callback);
 
-    indexBuffer->setBuffer(*engine, std::move(desc),
-            (uint32_t) destOffsetInBytes, (uint32_t) sizeInBytes);
```

Comment:
I assume sizeInBytes is computed and not passed?

Reply:
correct